Userland/Utilities/uptime.cpp

```diff
@@ -27,8 +28,6 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     auto file = TRY(Core::File::open("/sys/kernel/uptime"sv, Core::File::OpenMode::Read));
 
-    TRY(Core::System::pledge("stdio"));
```

Comment:
If you need additional permissions, add them, rather than removing `pledge()` entirely. I'm a bit rusty but I think the one you're missing is `rpath`, to read files, so `pledge("rpath stdio")`.

With that, we also want to maintain the security of not being able to read other arbitrary files. That means calling `Core::unveil()` for just the paths we want to read (so, `/var/run/utmp`) and then calling it again with null to lock the veil. You can find examples in other apps for how to do that.

Reply:
Okay I added the calls to `unveil()` (unveil is cool!)

Reply:
Regarding removing the `pledge()` - if you look above there is a more permissive pledge that is staying. I'm just removing this pledge because it removes the ability to read files (I'm not sure why).

If that second pledge is useful to keep, we could read `/var/run/utmp` before the call to that pledge.

Reply:
Oh, you need to also call `unveil(nullptr, nullptr)` afterwards to lock it, which prevents adding more paths.

I missed that there was another `pledge()` call! That should be fine, because the `unveil()`s already prevent reading other paths.

Reply:
Ah okay! Done 👍